test/test_bson_corpus.py

```diff
@@ -17,10 +17,17 @@
 import binascii
 import functools
 import glob
-import json
 import os
 import sys
 
+if sys.version_info[:2] == (2, 6):
+    try:
+        import simplejson as json
+    except ImportError:
+        import json
+else:
+    import json
+
```

Comment:
I know I use it often -- I find `python test/test_bson_corpus.py` nicer to type and to remember than `python setup.py test -s test.test_bson_corpus`.